iron-validatable-behavior.html

```diff
@@ -60,15 +70,24 @@
       this._validatorMeta = new Polymer.IronMeta({type: this.validatorType});
     },
 
+    _invalidChanged: function() {
+      if (this.invalid) {
+        this.setAttribute('aria-invalid', '');
```

Comment:
I think you need to set this to an actual value. The spec (http://www.w3.org/WAI/PF/aria/states_and_properties#aria-invalid) allows these values: `grammar`, `false`, `spelling`, `true`.